ts/IconButton/index.tsx

```diff
@@ -27,8 +27,10 @@ export class IconButton extends MaterialComponent<
 
   public componentDidMount() {
     super.componentDidMount();
-    this.MDComponent = new MDCIconButtonToggle(this.control);
-    this.MDComponent.listen('MDCIconButtonToggle:change', this.onChange);
+    if (this.control) {
```

Comment:
Why will this be missing?

Reply:
What will be missing, there's no "real" deletion it's just git being wired

Reply:
Why this.control in an if?

Reply:
@cromefire any issues?

Reply:
Not yet but there may be issues

Reply:
Cannot hurt

Reply:
extra code.